static/index.html

```diff
@@ -84,6 +84,56 @@ <h1 id="grapheneos">
             href="https://attestation.app/about">about page</a> and <a
             href="https://attestation.app/tutorial">tutorial</a>. These also support other
             operating systems.</p>
+
+            <h2 id="early-stage-of-development">
```

Comment:
This section got removed by the recent changes so you should remove it too:

https://github.com/GrapheneOS/grapheneos.org/commit/516afd143426e27dbc18fb1dd50ba24504f2f73d

I think you mixed up the merge conflict.

Reply:
Whoops! I'll try resetting this one and trying it again.

Reply:
I think I got it!